core/src/main/java/io/grpc/Attributes.java

```diff
@@ -29,8 +29,8 @@
 
 /**
  * An immutable type-safe container of attributes.
+ * @since 1.13.0
  */
-@ExperimentalApi("https://github.com/grpc/grpc-java/issues/1764")
```

Comment:
Shoot. I just saw noticed that hashCode does not agree with equals. Let's leave the annotation on this class a bit longer until we get that sorted out.